py2hwsw/lib/hardware/arith_logic/iob_int_sqrt/iob_int_sqrt.py

```diff
@@ -165,7 +165,7 @@ def setup(py_params_dict):
         assign a_in = {a[DATA_W-3:0], 2'b00};
         assign tmp =  r[SIZE_W+1] ? left + right : left - right;
         assign res_o = q;
-        assign done_o = ~pc0;
+        assign done_o = ~pc;
```

Comment:
what are the assignments ?

Reply:
can we do a comb? move back to iob-soc and make sure the tests pass

Reply:
In `iob_int_sqrt` I was just fixing that typo. Since my last PR fsms don't generate pcs with numbers

Reply:
There is an old version of `iob_int_sqrt` in iob-soc with a snippet.